vcert/connection_cloud.py

```diff
@@ -607,22 +622,122 @@ def set_policy(self, zone, policy_spec):
             raise VenafiError('User Details not found')
 
         app_details = self._get_app_details_by_name(app_name)
+        for_update, owners_list = self.resolve_owners(policy_spec.users, user_details)
         if app_details:
-            # Application exists. Update with cit
-            if not self._policy_exists(zone):
+            # Application exists. Update with cit and owners
+            if for_update and len(owners_list) > 0:
```

Comment:
Why an error is raised if the owners_list is bigger than zero? It looks counter-intuitive